roomdb/sqlite/roomconfig.go

```diff
@@ -68,3 +67,46 @@ func (c Config) SetPrivacyMode(ctx context.Context, pm roomdb.PrivacyMode) error
 
 	return nil // alles gut!!
 }
+
+// TODO: use proper language tag from "golang.org/x/text/language"?
```

Comment:
Good suggestion but lets make this a follow up issue for when we actually receive translations.